styles.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: red;
+}
```

Comment:
- [ ] Kindly use the class name suggested in the HTML file above to style your `h1` content in your CSS file.

E.g 👇 

```CSS
.heading {
  color: red;
}
```